test/SignatureChecker.t.sol

```diff
@@ -288,6 +428,30 @@ contract SignatureCheckerTests is TestUtils {
         assertTrue(_signatureChecker.isValidECDSASignature(account_, digest_, _encodeSignature(v_, r_, s_)));
     }
 
+    function test_eip2098SignatureIsNotMalleable() public {
+        (address alice_, uint256 privateKey_) = makeAddrAndKey("alice");
+
+        bytes32 digest_ = "TEST_DIGEST";
+        (uint8 v_, bytes32 r_, bytes32 s_) = vm.sign(privateKey_, digest_);
+        bytes memory signature_ = _encodeSignature(v_, r_, s_);
+
+        vm.prank(alice_);
+        _verifier.verifySignature(digest_, signature_);
+        assertEq(_verifier.signatureCounter(alice_), 1);
+
+        vm.prank(alice_);
+        vm.expectRevert(abi.encodeWithSelector(Verifier.SignatureUsed.selector, address(_verifier)));
```

Comment:
```suggestion
        vm.expectRevert(abi.encodeWithSelector(Verifier.SignatureUsed.selector, address(_verifier)));
        vm.prank(alice_);
```

Reply:
Same code?

Reply:
Yes, just usually I nitpick that `expect` before `prank`.